File: backEnd/platform-automation/src/main/java/com/mokatest/platform/demos/domain/ui/dto/step/UploadFileStepDTO.java
package com.mokatest.platform.demos.domain.ui.dto.step;

import com.mokatest.platform.demos.domain.ui.dto.ElementDTO;
import lombok.Data;

import java.util.List;

@Data
public class UploadFileStepDTO extends StepBaseDTO {
    private String stepType; // 步骤类型
    private ElementDTO element; // 文件输入框元素
    private List<String> fileIds; // 上传文件的fileId列表
}

File: frontEnd/src/types/dto/stepDTO/UploadFileStepDTO.ts
// 文件上传操作
import {StepBaseDTO} from "@/types/dto/base/StepBaseDTO";
import {ElementDTO} from "@/types/dto/ElementDTO";

export class UploadFileStepDTO extends StepBaseDTO {
    stepType: string = "FILE_UPLOAD"; // 步骤类型
    element: ElementDTO = new ElementDTO(); // 文件输入框元素
    fileIds: string[] = []; // 上传文件的fileId列表
}
